NavigationSystem/ASPire.mk

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+#
+# ASPire Makefile
+#
+###############################################################################
+
+
+###############################################################################
+# Files
+###############################################################################
+
+MAIN = main_ASPire.cpp
+OBJECTS = $(addprefix $(BUILD_DIR)/, $(SRC:.cpp=.o))
+
+HARDWARE_SERVICES_SRC = Hardwares/i2ccontroller/I2CController.cpp \
```

Comment:
I prefere to have the variables (group of files with their paths) declare in the main makefile. So the file paths are only written in one and same place (easier to change if necessary).
Then in each .mk file, those variables are used to define the source files to build.